components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -3442,4 +3432,80 @@
                 "EXPIRY_TIMESTAMP, TENANT_ID, TOKEN_TYPE) VALUES(?,?,?,?,?)";
         public static final String DELETE_REVOKED_JWT = "DELETE FROM AM_REVOKED_JWT WHERE EXPIRY_TIMESTAMP < ?";
     }
+
+    //Shared Scopes related constants
+    public static final String ADD_SHARED_SCOPE = "INSERT INTO AM_SHARED_SCOPE (NAME, UUID, TENANT_ID) VALUES (?,?,?)";
+    public static final String DELETE_SHARED_SCOPE = "DELETE FROM AM_SHARED_SCOPE WHERE NAME = ? AND TENANT_ID = ?";
+    public static final String GET_SHARED_SCOPE_BY_UUID = "SELECT * FROM AM_SHARED_SCOPE WHERE UUID = ?";
+    public static final String GET_ALL_SHARED_SCOPES_BY_TENANT = "SELECT * FROM AM_SHARED_SCOPE WHERE TENANT_ID = ?";
```

Comment:
shall we use specific fields instead of *?

Reply:
+1 Fixed with https://github.com/wso2/carbon-apimgt/pull/8409/commits/0f586a5447b97cc79c01a0ff3a4a7a94a1ce74cf